Workbooks/TSG/Performance Counter TSG/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Performance Counter TSG",
+    "author": "Microsoft",
+    "galleries": [{ "type": "tsg", "resourceType": "microsoft.insights/components", "order": 100 }]
```

Comment:
Why not type == 'Troubleshooting Guide'?

Reply:
This is a key that won't be shown to an user